src/py21cmfast/wrapper.py

```diff
@@ -2415,3 +2448,80 @@ def _get_lightcone_redshifts(
             for d in lc_distances
         ]
     )
+
+def _CalibratePhotonCons(
```

Comment:
can we rename this `calibrate_photon_cons`?

Reply:
Done